cliboa/scenario/extract/gcp.py

```diff
@@ -91,15 +91,16 @@ def _save_to_cache(self):
             key_filepath = self._credentials
         else:
             key_filepath = self._source_path_reader(self._credentials)
-        df = pandas.read_gbq(
-            query="SELECT * FROM %s.%s" % (self._dataset, self._tblname)
-            if self._query is None
-            else self._query,
-            dialect="standard",
+
```

Comment:
Would it be possible to use the method below for getting BigQuery Client?
https://github.com/BrainPad/cliboa/blob/master/cliboa/util/gcp.py#L44

Reply:
Thank you for the review.
Replaced with above method (Needed to change parameter setting for it to work)

Reply:
@bo-naoki-okamoto